vnext/ReactCommon/TEMP_UntilReactCommonUpdate/react/renderer/components/view/ViewEventEmitter.cpp

```diff
@@ -156,4 +156,9 @@ void ViewEventEmitter::onKeyDown(KeyboardEvent const &event) const {
     });
 }
 
+// [Windows]
+void ViewEventEmitter::onClick() const {
```

Comment:
Do we even need onClick when Pressability is a thing now? https://reactnative.dev/docs/pressable

Reply:
Copying note from above: The value of props->onClick matches whether onPress has been set. And the call to dispatchEvent("topClick") runs the onPress function. I haven't dug into the code to figure out why this is the case, but that's the behavior we get.

So we don't need to pass any additional props through on the JS side to get onClick data. This code is just to allow us to access the onPress data on the native side.